test/integration/tasks_test.rb

```diff
@@ -120,18 +120,18 @@ def rake(cmd, options = {})
     env = {
       "PATH_TO_HIT" => 'foo_secret',
       "TEST_COUNT" => "2",
-      "HTTP_AUTHORIZATION" => "Basic #{Base64.encode64("admin:secret")}",
+      "HTTP_AUTHORIZATION" => "Basic #{Base64.strict_encode64("admin:secret")}",
```

Comment:
prior version unintentionally included a trailing linebreak which results in an invalid http header